sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -190,23 +207,39 @@ export class ServiceBusClient {
   ): ServiceBusReceiver<ReceivedMessage> | ServiceBusReceiver<ReceivedMessageWithLock> {
     // NOTE: we don't currently have any options for this kind of receiver but
     // when we do make sure you pass them in and extract them.
-    const { entityPath, receiveMode } = extractReceiverArguments(
+    const { entityPath, receiveMode, options } = extractReceiverArguments(
       queueOrTopicName1,
       optionsOrSubscriptionName2,
       options3
     );
 
+    let entityPathWithSubQueue = entityPath;
```

Comment:
This code looks good - we might as well just put it in extractReceiverArguments at this point.

Reply:
`extractReceiverArguments` applies to both "normal" and "session" receivers
But subQueue does not apply to session receivers.
So, decided to keep it here rather than in `extractReceiverArguments`